_episodes/11-using-arrays-transformations.md

```diff
@@ -21,31 +21,28 @@ Arrays can be sorted, de-duplicated, and manipulated in other ways in GREL expre
 
 "Monday,Tuesday,Wednesday,Thursday,Friday,Saturday,Sunday"
 
-This can be transformed into an array using the ```split``` function
+This can be transformed into an array using the ```split``` function with pipe
 ```
-value.split(",")
+value.split("|")
 ```
-This would create the array containing the days of the week:
+This would create the array containing a list of subject headings, separated by a pipe character | (as in the data)
 
-["Monday","Tuesday","Wednesday","Thursday","Friday","Saturday","Sunday"]
+For example:
 
-This can be combined with array operations like ```sort```. For example, assuming the cell contains the same value as above, then the function
-```
-value.split(",").sort()
-```
-would result in an array containing the days of the week sorted in alphabetical order:
+"crystal structure|clozapinium|crystal structure|molecular configuration|hydrogen bonding|supramolecular assembly|Chemistry|QD1-999"
 
-["Friday","Monday","Saturday","Sunday","Thursday","Tuesday","Wednesday"]
+This subject string can be found for the title "The crystal structures of three clozapinium salts: different molecular configurations, and supramolecular assembly in one, two and three dimensions" in the original project. 
 
-To output a value from an array you can either select a specific value depending on its position in the list (with the first position treated as 'zero'). For example
+This can be combined with array operations like ```uniques```. For example, assuming the cell contains the same value as above, then the function
```

Comment:
Reinstate the information about extracting a particular item from the array with the square bracket notation: value.split("|")[0]

Reply:
Using the subject heading example instead of days of the week